chapter_04/p09_bst_sequences.py

```diff
@@ -1,5 +1,7 @@
 from chapter_04.binary_search_tree import BinarySearchTree
 
+from collections import deque
+
```

Comment:
> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
Imports should be grouped in the following order:
> - Standard library imports.
> - Related third party imports.
> - Local application/library specific imports.
>
> You should put a blank line between each group of imports.


https://www.python.org/dev/peps/pep-0008/#imports

Reply:
```python
from collections import deque

from chapter_04.binary_search_tree import BinarySearchTree
```

Reply:
Thanks, I have made the corresponding changes.